src/demo/samples.json

```diff
@@ -5,6 +5,10 @@
   {
     "groupName": "General",
     "children": [
+      {
```

Comment:
it probably shouldn't be the first element in the list

Reply:
@targos I moved it to the list's end